R/helper.R

```diff
@@ -19,10 +19,25 @@ as_backend = function(id) {
 # @return c(measure_score_1, measure_score_2)
 binary_measure_score = function(prediction, base_measure, data_task){
   subcol = data_task$col_roles$pta
-  prediction = split(as.data.table(prediction), data_task$data(cols = subcol))
+  prediction = split(as.data.table(prediction), data_task$data(cols = subcol, rows = prediction$row_ids)[[1]])
   prediction = map(prediction, as_prediction_classif)
   msr1 = prediction[[1]]$score(base_measure)
   msr2 = prediction[[2]]$score(base_measure)
 
   return(c(msr1, msr2))
 }
+
+# Get Measures Name
+# @description
+# This helper function returns name of msr or msrs
+#
+# @param object one single measure (msr) or multiple measures (msrs)
+get_msrs_name <- function(object){
+  if( any(class(object) == "Measure") ) { return(object$id) }
+
+  result = rep(NA, length(object))
+  for(i in c(1:length(object))){
```

Comment:
`map_chr(object, function(x) ifelse(is.null(x$id, NA_character, x$id))`

Reply:
Solved